article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -28,6 +29,11 @@
         @if(model.currentPage.currentPage.isArchivePage) {
             @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage, amp = true)
         }
+
+            <script type="application/ld+json">
```

Comment:
I think you're most definitely right, I've put it in the wrong place. Will double check and look at fixing shortly!